CheatSheets/git-cheatsheet.md

```diff
@@ -161,5 +161,9 @@ Table of Contents
 |`git stash pop`| write working from top of stash stack|
 |`git stash drop` | discard the changes from top of stash stack|
 
+## Merge Unrelated histories
+| command | Description |
+| ------- | ----------- |
+| `git merge --allow-unrelated-histories`| Merge unrelated histories|
```

Comment:
```suggestion
| `git merge --allow-unrelated-histories` | Merge unrelated histories - Allow merging branches with no common history base |
```
Please, add this command in the `Branching & Merging` section.

Reply:
Alright I'll do that thanks for letting me know